common/types/offer.go

```diff
@@ -228,3 +222,19 @@ func (o *Offer) UnmarshalJSON(data []byte) error {
 	}
 	return o.validate()
 }
+
+// OfferExtra represents extra data that is passed when an offer is made.
+type OfferExtra struct {
+	// UseRelayer forces the XMR maker to claim using the relayer even when he
+	// has enough funds to make the claim himself. Setting it to false will not
+	// prevent the relayer from being used if there are insufficient ETH funds
+	// to claim.
+	UseRelayer bool `json:"useRelayer,omitempty"`
+}
+
+// NewOfferExtra creates an OfferExtra instance
+func NewOfferExtra(forceUseRelayer bool) *OfferExtra {
+	return &OfferExtra{
+		UseRelayer: forceUseRelayer,
+	}
+}
```

Comment:
I was doing custom marshaling to ensure that the `StatusCh` was always initialized in an early version of this PR. Otherwise I wouldn't have made so many edits here. Everyone is calling `NewOfferExtra` now instead of creating the structure directly.